includes/controllers/class.llms.controller.certificates.php

```diff
@@ -2,47 +2,83 @@
 /**
  * Certificate Forms
  *
- * @since   3.18.0
- * @version 3.24.0
+ * @since 3.18.0
+ * @version [version]
  */
 defined( 'ABSPATH' ) || exit;
 
 /**
  * @since 3.18.0
  * @since 3.35.0 Sanitize `$_POST` data.
+ * @since [version] Modify `llms_certificate` post type registration to allow certificate templates to be exported.
+ *                When exporting a certificate template, use the `post_author` for the certificate's WP User ID.
  */
 class LLMS_Controller_Certificates {
 
 	/**
 	 * Constructor
 	 *
-	 * @return   void
-	 * @since    3.18.0
-	 * @version  3.18.0
+	 * @since 3.18.0
+	 * @since [version] Add filter hook for `lifterlms_register_post_type_llms_certificate`.
+	 *
+	 * @return void
 	 */
 	public function __construct() {
 
+		add_filter( 'lifterlms_register_post_type_llms_certificate', array( $this, 'maybe_allow_public_query' ) );
+
 		add_action( 'init', array( $this, 'maybe_handle_reporting_actions' ) );
 		add_action( 'wp', array( $this, 'maybe_authenticate_export_generation' ) );
 
 	}
 
 	/**
-	 * Utilizes a nonce to display a certificate
+	 * Modify certificate post type registration data during a certificate template export.
+	 *
+	 * Fixes issue https://github.com/gocodebox/lifterlms/issues/776
+	 *
+	 * @since [version]
+	 *
+	 * @param array $post_type_args Array of `llms_certificate` post type registration arguments.
+	 * @return array
+	 */
+	public function maybe_allow_public_query( $post_type_args ) {
+
+		if ( ! empty( $_REQUEST['_llms_cert_auth'] ) ) { // phpcs:disable WordPress.Security.NonceVerification.Recommended
+
```

Comment:
I think would be a good practice to re-enable the phpcs rule disabled in the line above.

Reply:
Good catch, I meant to use `phpcs:ignore` which allows you to ignore the single line instead of disabling the rule for the whole file.

Reply:
This makes me realize that I've made this mistake quite a bit throughout the codebase... Going to open a new issue to address that.